### DISCUSSION TITLE
Where are the docker images for Kong 3.2.X?

### DISCUSSION THREAD

**Discussion by peter-dough** [Kong/kong#10554] [Help] [answered]
The releases shown [here](https://github.com/Kong/kong/releases) suggests that there are docker images paired with each release. I think this is generally true, but when looking at the Docker page for [kong barebones](https://hub.docker.com/_/kong), the 3.2.X tags are not listed. Is this something that was missed? Or perhaps in progress? Or should I be looking elsewhere? Please advise.

**Comment by guanlan** [comment, marked as answer]
https://hub.docker.com/_/kong/tags
3.2 tags are shown, can you check again?

**Reply by peter-dough** [reply]
Awesome, thanks for the update. Looks like they were just uploaded one day after the original post.
